src/core/pointcloud/qgspointcloudlayer.h

```diff
@@ -212,6 +219,19 @@ class CORE_EXPORT QgsPointCloudLayer : public QgsMapLayer
     bool convertRenderer3DFromRenderer2D();
 
 
+    /**
+     * Returns the statistic \a statistic of \a attribute
+     *
+     * \since QGIS 3.26
+     */
+    QVariant statistic( const QString &attribute, QgsStatisticalSummary::Statistic statistic ) const;
+
+    /**
+     * Returns whether the point cloud layer contains calculated statistics
+     *
+     * \since QGIS 3.26
+     */
+    bool hasCalculatedStatistics() const { return mStatisticsCalculated; }
```

Comment:
If we can return only true/false, then I guess we are unable to tell whether statistics are being calculated, or if they do not need to be calculated...

I think it would be better to have enum, similar to the one for indexing where we would be able to indicate the state of statistics more precisely, e.g.:
- "not started"
- "calculating"
- "calculated"

This may be useful for some third party code...